### PR TITLE
Add board.LED for Grand Central M4 Express

### LINKED ISSUE #5111: board.LED not defined for Grand Central M4 Express
### CircuitPython version

```python
Adafruit CircuitPython 7.0.0-alpha.5-140-g9a6b11806 on 2021-08-08; Adafruit Grand Central M4 Express with samd51p20
```

### Code/REPL

```python
import board
from digitalio import DigitalInOut, Direction
led = DigitalInOut(board.LED)
```

### Behavior

The following error is displayed:

```
AttributeError: 'module' object has no attribute 'LED'
```

### Description

Although this is a minor issue, for code portability it would be nice not to have to use conditional logic or to change the original source code.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by NathanY3G**
Fixes #5111

### CODE REVIEW

**Comment by NathanY3G** [comment]
Tested on a GCM4 by running the following code from the REPL.

```
import board
from digitalio import DigitalInOut, Direction
led = DigitalInOut(board.LED)
led.direction = Direction.OUTPUT
led.value = True
```

File: ports/atmel-samd/boards/grandcentral_m4_express/pins.c [hunk 1]
```diff
@@ -137,6 +137,7 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
 
     { MP_OBJ_NEW_QSTR(MP_QSTR_NEOPIXEL), MP_ROM_PTR(&pin_PC24) },
 
+    { MP_OBJ_NEW_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_PB01) },
```
Comment (dhalbert) [inline ports/atmel-samd/boards/grandcentral_m4_express/pins.c:140]: Please move this up to where `D13` is defined, and put blank lines around D13 and LED, to indicate that it's a pair of aliases. Thanks.

**Review by dhalbert — CHANGES_REQUESTED** [review]
Thanks for doing this. I missed it in my big sweep in #5066.

File: ports/atmel-samd/boards/grandcentral_m4_express/pins.c (hunk 1 above)
Comment (NathanY3G) [reply ports/atmel-samd/boards/grandcentral_m4_express/pins.c]: Done

**Comment by NathanY3G** [comment]
> Thanks for doing this. I missed it in my big sweep in #5066.
I am glad I could help out :-)

**Review by dhalbert — APPROVED** [review]

**Pull request merged**